content/static/stylesheets/default.sass

```diff
@@ -130,6 +130,23 @@ img
       .ui.header a
         color: inherit
 
+/* Position, width, margin modifications for right rail
+/* Only for screen wider than 1200px
+.right.rail
+  left: 827px!important
+  .header
+    margin-top: 3em!important
+
+.ui.demo.container
+  margin-left: calc((100% - 1127px)/2) !important
+  margin-right: 387px!important
+  .tab
+    width: 827px
+    .example:last-child:not(.position)
```

Comment:
Yes, I set the padding-bottom of last example to 35em, to keep the "transition" item in right rail become "active" and change color. To make it more comfortable, I may apply some more specific definitions on the last example.